server/app.js

```diff
@@ -22,3 +22,4 @@ app.get("/home/:search", (req, res) => {
 app.listen(port, () => {
   console.log(`Server running on port ${port}`);
 });
+
```

Comment:
One thing I would change is separating the app.listen() into its separate file as it if you were to run tests you need to create your own mock server to test posts and get requests inside of the app.js file you don't want both servers to clash.

Reply:
Also when sending errors its good to send the status code of the error.